qt5/platforminputcontext/qfcitxplatforminputcontext.cpp

```diff
@@ -482,6 +481,10 @@ void QFcitxPlatformInputContext::createInputContextFinished(
     }
     flag |= FcitxCapabilityFlag_ClientSideInputPanel;
 
+    if (!inputMethodAccepted() || !objectAcceptsInputMethod()) {
```

Comment:
also need to update the flag when it changes from disable to enable, vice versa.

need add some code in ::update

Reply:
done